packages/mds-event-processor/src/annotation.ts

```diff
@@ -0,0 +1,22 @@
+import { AnnotationData, GpsData } from '@mds-core/mds-types'
+import { findServiceAreas } from './geo/geo'
+
+const version = 1.0
```

Comment:
Just curious, what does this version number come from? It's different from the version number declared in the package.json.

Reply:
I believe this is the `annotation` version number, which is independent from the package version.